e2e/fixtures/events/createClaim.js

```diff
@@ -92,6 +98,12 @@ const createClaimData = (legalRepresentation, useValidPba) => {
         },
       },
     },
+    DefendantSolicitorServiceAddress: {
```

Comment:
We should also include `SecondDefendantSolicitorServiceAddress` page in api tests. See how it's done for `AddAnotherClaimant`.